### ISSUE TITLE
EVSS PCIU Migration: Step 6a: Remove PCIU references from pre-fill 

### ISSUE THREAD

**Issue by RachalCassity** [department-of-veterans-affairs/va.gov-team#76166]
## User Story
As a engineer on the VA.gov Platform, working on the PCIU deprecation, I want to remove PCIU, so that PCIU is completely removed from form prefill functionality

## Issue Description

The EVSS PCIU service is being deprecated and removed from Vets API. PCIU references within the pre-fill logic need to be removed. This cleanup will streamline the pre-fill process and ensure the codebase reflects the current state of services.

## Tasks
- [ ] ~Ensure Feature Toggle in OFF for PCIU and ON for VA Profile Contact Information in prod~
- [x] Remove PCIU from app/models/user.rb
- [x] Remove PCIU from app/models/form_profile.rb
- [x] Remove PCIU from spec/models/form_profile_spec.rb
- [x] Remove PCIU from spec/models/user_spec.rb
- [x] Remove PCIU from app/models/form_profiles/va_526ez.rb
- [x] Remove flipper references: remove_pciu, remove_pciu_2
- [x] Ensure tests are passing
- [ ] ~Test locally~
- [ ] ~Test in staging~

## Acceptance Criteria
- [ ] ~PCIU is removed from pre-fill logic~
- [ ] PR is created and is ready for review after all blockers are removed.

**Comment by jennb33** [comment]
Per @RachalCassity, she has been working with Seth Darr. PCIU calls have been removed and implementation has been held up. Disability Benefits team is focused on the the 526 form, and it doesn't appear that we are overlapping.

**Comment by RachalCassity** [comment]
@jennb33 Moving this back to the Upcoming Sprint. Waiting for Benefits team to update all PCIU. This service is still being used, datadog pciu metrics.

**Comment by RachalCassity** [comment]
Contact Information V2 has to replace PCIU in prod before PCIU references can be removed.

**Comment by AshleyGuerrant** [comment]
@RachalCassity - thanks for the additional context as to why this ticket is blocked.

> Contact Information V2 has to replace PCIU in prod before PCIU references can be removed.

What specific ticket numbers(s) are tracking the work mentioned above & are blocking this this work?

**Comment by AshleyGuerrant** [comment]
This work is blocked by [Step 4c](https://github.com/department-of-veterans-affairs/va.gov-team/issues/99322).

**Comment by RachalCassity** [comment]
@AshleyGuerrant Started working on this on 12/26

**Comment by RachalCassity** [comment]
Pull request is waiting in progress until mobile upgrades Contact Information from V1 to V2. All tests besides mobile are passing.

**Comment by AshleyGuerrant** [comment]
**Sprint reporting note:** this ticket is blocked until the mobile testing is complete. Moving out to Sprint 20, but I will comment & unblock this ticket if the mobile testing happens to get completed during Sprint 19.

**Comment by RachalCassity** [comment]
Mobile pull request was merged yesterday. This work is no longer blocked and should be added to the next sprint.

**Comment by AshleyGuerrant** [comment]
**Sprint reporting note:** This ticket will roll over to Sprint 22, because it was blocked until today (2/25/25). Mobile [testing was complete](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97889#issuecomment-2654702670) as of 2/12/25. However, the VFS teams testing was just fully completed today, now that #98925 is done.

**Comment by RachalCassity** [comment]
[Pullrequest](https://github.com/department-of-veterans-affairs/vets-api/pull/21041/commits) is ready for review but cannot be merged until Contact Info V2 replaces PCIU in prod.

**Comment by AshleyGuerrant** [comment]
PR is ready but cannot merge until [Step 5.5](https://github.com/department-of-veterans-affairs/va.gov-team/issues/101375) is complete. Moving this ticket to blocked.

**Comment by AshleyGuerrant** [comment]
**Sprint Planning note:** @RachalCassity is going to write up a separate ticket to track the PR merge for this work. The majority of this work is already complete, but we are blocked on merging the PR until the[ VAProfile API key is obtained](https://github.com/department-of-veterans-affairs/va.gov-team/issues/103958).
